doc/src/manual/cowboy_rest.asciidoc

```diff
@@ -99,11 +102,15 @@ For PUT requests, the body is a representation of the resource
 that is being created or replaced.
 
 For POST requests, the body is typically application-specific
-instructions on how to process the request, but it may also
-be a representation of the resource. When creating a new
-resource with POST at a different location, return `{true, URI}`
+instructions on how to process the request, but it may also be a
+representation of the resource. When creating a new resource with POST
+at a different location, return `{created, URI}` or `{see_other, URI`}
 with `URI` the new location.
 
+Returning `{created, URI`} will result in a '201 Created' response.
+
+Returning `{see_other, URI`} will result in a '303 See Other' response.
+
```

Comment:
That's not what I meant. The exact status code is not important here (it will be important in the updated diagrams). What's important is to say that the `see_other` return value implies a redirect. Something like

```
The `see_other` tuple will redirect the client to the new location automatically.
```

at the end of the paragraph.

Reply:
I tried to follow the style of other callbacks; the first one I found was delete_completed which has:
```
Returning `false` will result in a '202 Accepted' response
```
I will change this to the sentence you suggested.

Reply:
Right the existing manual isn't very consistent in that regard.